NEWS.txt

```diff
@@ -42,13 +42,23 @@ restore snapshots created with the previous major version using the
 'sstableloader' tool. You can upgrade the file format of your snapshots
 using the provided 'sstableupgrade' tool.
 
-3.0.21
+3.11.9
+======
+Upgrading
+---------
+   - Custom compaction strategies must handle getting sstables added/removed notifications for
+     sstables already added/removed - see CASSANDRA-14103 for details. This has been a requirement
+     for correct operation since 3.11.0 due to an issue in CompactionStrategyManager.
+
+3.11.7
+>>>>>>> aef2d0f7ca... Review feedback
```

Comment:
@ekaterinadimitrova2 Looks like a merge error here?

Reply:
Yes and CI can't catch it, I am sorry I missed it. The plan was to remove the whole entry actually (referring to your other question in 3.11,4), done already in the other branches and added to the rst doc as suggested in the ticket